guides/hosting/performance/cart-storage.md

```diff
@@ -16,11 +16,33 @@ Redis is better suited in high-throughput scenarios, therefore you should use Re
 
 To use Redis, create a `config/packages/shopware.yml` file with the following content:
 
+<Tabs>
+<Tab title="Before v.6.6.8.0">
```

Comment:
why don't you change the point after the `v`? :sweat_smile: it really bothers me :grin: is it needed?

Reply:
Lol, sorry, was making changes in IDE and totally missed this. I hope now it's good.